src/services/ieq/ieq.js

```diff
@@ -387,25 +387,29 @@ IEQ.prototype.closeSubscriptions = function(){
 
 /**
 * Request Data to make CSV file
-	* @param {list} sensorNames : list of sensor and index names
-	* @param {number} _firstDay: timestamp of beginning time
-  	* @param {string} timeSample: timeinterval for data. Parameters: "second", "minute", "hour", "day", "week", "month"
-	* @param {number} _nlines: maximum number of lines requested
-	* @param {callback} callback: called after update
+	 @param {object} csvConfig params:
+		* @param {list} sensorNames : list of sensor and index names
+		* @param {number} _startTime: timestamp of beginning time
+		* @param {number} _endTime: timestamp of end time
+	  	* @param {string} timeSample: timeinterval for data. Parameters: "second", "minute", "hour", "day", "week", "month"
+		* @param {number} _nlines: maximum number of lines requested
+		* @param {callback} callback: called after update
 */
```

Comment:
Not sure that this is the correct way to signify that list, etc. are fields of csvConfig.
callback indentation seems wrong.

Good point for the documentation, it looks serious, but I will insist on doing it properly to avoid having to correct it later.